mlx/directives/item_relink_directive.py

```diff
@@ -49,7 +49,12 @@ def apply_effect(self, collection):
             item.remove_targets(source_id, explicit=True, implicit=True, relations={forward_type})
             source.remove_targets(item_id, explicit=True, implicit=True, relations={reverse_type})
             if target_id:
-                collection.add_relation(item_id, forward_type, target_id)
+                try:
+                    collection.add_relation(item_id, forward_type, target_id)
+                except TraceabilityException as err:
+                    if not self['nooverwrite']:
+                        report_warning(err, self['document'], self['line'])
```

Comment:
This is behavior is new. Instead of raising `mlx.traceability_exception.TraceabilityException: duplicating <<new_source>> passes <<target>>`, you will report a warning. I consider this an improvement instead of a backwards incompatible change. :+1: